admin/resources/js/adminRoutes.ts

```diff
@@ -1,57 +1,71 @@
+import path from "path-browserify";
 import { ADMIN_APP_DIR } from "./adminConstants";
 
-export const homePath = (): string => `/${ADMIN_APP_DIR}`;
+export const homePath = (): string => path.join("/", ADMIN_APP_DIR); // leading slash in case empty base url
 export const homePublicPath = (): string =>
-  `/${ADMIN_APP_DIR}/public/${ADMIN_APP_DIR}`;
```

Comment:
I feel this original line with `ADMIN_APP_DIR` twice was a bug.  Was I correct to remove the second instance of it?

Reply:
I think it was more of a workaround than a bug - our htaccess used to redirect you to /admin/public/admin sometimes. Happily, we've resolved those issues. We should be able to remove `homePublicPath` entirely, actually.

Reply:
Ah, that makes sense.  OK, I've gone ahead and removed the constant entirely and routing seems to still work fine on the admin site.